client/js/config/constant.js

```diff
@@ -1 +1,5 @@
+export const CharityCategory = '../assets/category/Charity.png';
+export const EnviroCategory = '../assets/category/Enviro.png';
+export const PoliticoCategory = '../assets/category/Politico.png';
+export const SocialCategory = '../assets/category/Social.png';
```

Comment:
If you want to require images, you should require them where you need them. require can not take variable as an input.

Reply:
Done. I've deleted the 4 image consts.